docs/workforce-management-framework/topics/con-raincatcher-core-getting-started.adoc

```diff
@@ -0,0 +1,143 @@
+[[id='raincatcher-core-getting-started-{chapter}']]
+= RainCatcher-core Getting Started
+
+Getting started with the Feedhenry RainCatcher Core Repository
```

Comment:
Looks like the README is out of date . Need a single source of truth for all docs.